styles/editMember.css

```diff
@@ -0,0 +1,209 @@
+#editMemberform {
+
+    margin-left: 145px;
+    margin-top: 48px;
+      
+}
+
+body{
+    background-color: black;
+}
+
+h2 {
+
+width: 198px;
+height: 29px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 600;
+font-size: 24px;
+color: #FFFFFF;
+
+}
+
+#fName {
+
+width: 83px;
+height: 19px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 500;
+font-size: 16px;
+color: #FFFFFF;
+
+}
+
+#firstname{
+
+box-sizing: border-box;
```

Comment:
I told you that copying the styles directly from Figma won't work because they're auto-generated without context of what our entire app is doing! Ask Kratti to help you with this one